site-modules/adhoc/tasks/win_example.json

```
@@ -0,0 +1,4 @@
+{
+     "description": "This is the description for the win_example task",
+     "input_method": "powershell"
+   }
```

Indentation for JSON: 4 spaces, closing brace at same level as opening brace
```suggestion
    "description": "This is the description for the win_example task",
    "input_method": "powershell"
}
```